From 3c951fd7f503e01c350b7af3aca11af1aff2e6a9 Mon Sep 17 00:00:00 2001
From: Helio Chissini de Castro <helio@kde.org>
Date: Tue, 27 Sep 2016 19:48:20 +0200
Subject: [PATCH 0/4] - Fix endif placement when test for MATH library.

The wrong placement of the test is breaking compilations that do not use
muParser library
---
 providers.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/providers.cpp b/providers.cpp
index 2e975a2..896a33f 100644
--- a/providers.cpp
+++ b/providers.cpp
@@ -885,6 +885,8 @@ MathProvider::MathProvider()
     append(new MathItem());
 }
 
+#endif
+
 ExternalProviderItem::ExternalProviderItem()
 {
 }
@@ -974,4 +976,3 @@ void ExternalProvider::readFromProcess()
     }
 }
 
-#endif
-- 
2.8.3

